bson/src/main/org/bson/conversions/Bson.java

```diff
@@ -63,7 +65,8 @@
                     new MapCodecProvider(),
                     new Jsr310CodecProvider(),
                     new JsonObjectCodecProvider(),
-                    new BsonCodecProvider()));
+                    new BsonCodecProvider(),
+                    new EnumCodecProvider()));
```

Comment:
As far as I can see there are no regression tests for the default registries, so I didn't add new tests.